### ISSUE TITLE
Fix sort order for Play From Here

### ISSUE THREAD

**Issue by ScreamingHawk** [ScreamingHawk/android-slideshow#118]
The `Play From Here` button currently sorts alphabetically by filename when it should sort alphabetically by path. 

This should be an easy fix by changing this line: https://github.com/ScreamingHawk/android-slideshow/blob/master/app/src/main/java/link/standen/michael/slideshow/util/FileItemHelper.java#L68

e.g.

```
a/
  1.jpg
  2.jpg
b/
  1.jpg
  2.jpg
c/
  1.jpg
  2.jpg
```

Current:
```
a/1.jpg
b/1.jpg
c/1.jpg
a/2.jpg
b/2.jpg
c/2.jpg
```

Expected:
```
a/1.jpg
a/2.jpg
b/1.jpg
b/2.jpg
c/1.jpg
c/2.jpg
```

**Comment by brandanlockwood** [comment]
Can I work on this?

**Comment by ScreamingHawk** [comment]
Absolutely! Thanks so much

Let me know if you've got any questions and I can give you a hand.

Cheers,
Michael

**Comment by brandanlockwood** [comment]
Hi Michael,
At https://github.com/ScreamingHawk/android-slideshow/blob/master/app/src/main/java/link/standen/michael/slideshow/model/FileItem.java#L59 is there reason to compare names versus paths?

**Comment by ScreamingHawk** [comment]
Not really. 

Originally there was only a single folder to slideshow from, so the `path` part of the `path` was always the same. This fix should be as easy as changing that line as you've pointed out.

**Comment by ScreamingHawk** [comment]
Fixed in #119
